wagtail/images/fields.py

```diff
@@ -17,6 +17,34 @@ def get_allowed_image_extensions():
         ["avif", "gif", "jpg", "jpeg", "png", "webp"],
     )
 
+def image_format_name_to_content_type(image_format_name):
+    """
+    Convert a Willow image format name to a content type using the mime_type property.
```

Comment:
I think the original intention of the TODO comment was to completely remove this utility function in favour of accessing the image file instance's `mime_type` property where this util function was originally used, so I'm not convinced this is the correct approach.

I haven't checked how feasible that is in the context of the code that used this function, though.